testutils/inject/digital_interrupt.go

```diff
@@ -60,3 +61,12 @@ func (d *DigitalInterrupt) AddCallback(c chan board.Tick) {
 	}
 	d.AddCallbackFunc(c)
 }
+
+// RemoveCallback calls the injected AddCallback or the real version.
```

Comment:
Also migrate to take in a board it can stay and be in the inject package same was as New* is used from these utils